test/unit/loggerTest.ts

```diff
@@ -157,6 +157,17 @@ describe('Logger', () => {
       expect(Logger.root['called']).to.be.true;
       expect(rootLogger.addLogFileStream['called']).to.be.false;
     });
+
+    it('should log uncaught exception in root logger', async () => {
+      process.env.SFDX_ENV = 'dev';
+
+      const rootLogger = await Logger.root();
+      $$.SANDBOX.stub(rootLogger, 'fatal');
+
+      // @ts-ignore
```

Comment:
could you please add the reason for turning off TS compilation.  something like, "to access private properties for testing" or however you'd like to word it.  In the child test as well.  Thanks!

Reply:
Sure, will do. Thanks for the feedback